workflows/database_upsert.go

```diff
@@ -103,7 +103,10 @@ func (workflow *databaseWorkflow) databaseDeployer(namespace string, service *co
 		}
 
 		//DatabaseMasterPassword:
-		dbPass, _ := paramManager.GetParam(fmt.Sprintf("%s-%s", dbStackName, "DatabaseMasterPassword"))
+		dbPass, err := paramManager.GetParam(fmt.Sprintf("%s-%s", dbStackName, "DatabaseMasterPassword"))
+		if err != nil {
```

Comment:
I don't think this `err` should be returned...rather it should be ignored.  If the DB master password isn't set yet, then it is generated.  Maybe just replace with an info message that it didn't exist so about to generate one?